src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -730,23 +732,6 @@
     @Nonnull
     public ItineraryFilterParameters itineraryFilters = ItineraryFilterParameters.createDefault();
 
-    /**
-     * The numbers of days before the search date to consider when filtering trips for this search.
-     * This is set to 1 to account for trips starting yesterday and crossing midnight so that they
-     * can be boarded today. If there are trips that last multiple days, this will need to be
-     * increased.
-     */
-    public int additionalSearchDaysBeforeToday = 1;
-
-    /**
-     * The number of days after the search date to consider when filtering trips for this search.
-     * This is set to 1 to account for searches today having a search window that crosses midnight
-     * and would also need to board trips starting tomorrow. If a search window that lasts more than
-     * a day is used, this will need to be increased.
-     */
-    public int additionalSearchDaysAfterToday = 2;
-
```

Comment:
I think these two parameters should be replaced by a parameter for the:

```java
/** 
 * The expected maximum time a journey can last across all possible journeys for the current deployment. 
 * Normally you would just do an estimate and add enough slack so you are sure that there is no journeys that
 * falls outside this window. The parameter is used find all possible dates for the journey and then search only
 * the services witch run on those dates. The duration must include access, egress, wait-time and transit time
 * for the hole journey. It should also take low frequency days/periods into account like holidays. In other words,
 * pick the to points within your area that has the worst connection and then try to travel on the worst possible
 * day, and find the maximum journey duration. Using a value that is too high have the effect of including more
 * patterns in the search; Hence, making it a bit slower. Recommended values would be from 12 hours(small
 * town/city), 1 day (region) to 2 days (country like Norway).
 */
public Duration maxJourneyDuration = Duration.ofDays(1);
```

Reply:
You also need to add support for Duration in in the `NodeAdaptor`, I used an array of duration in PR [Adjust search window dynamically to fit the number of itineraries #3828](https://github.com/opentripplanner/OpenTripPlanner/pull/3828/files). So you do not need to add doc for the new type, just the new parameter. 
```java
public Duration asDuration(String paramName, Duration defaultValue) {
       if(!exist(paramName)) { return defaultValue; }
       return DurationUtils.duration(asText(paramName));
}
```

Reply:
Should we use the term "Journey" internally in OTP to represent going from the origin to the destination place, and "Trip" to alway be a ServiceJourney/GTFS Trip?

Reply:
Also, `maxJourneyDuration` could be used to set latest arrival time, which could improve performance, by pruning alternatives

Reply:
Yes in teori it can, but I think it is a big refactoring. I think quite a bit of code now assume that the LAT is not set for a depart-after search, and setting a LAT so late that most patterns are not running any more will not have a huge performance impact.

Reply:
#3838 only implements parsing a list of durations. Would you be ok if I adding parsing a single duration which is what I need here?